### ISSUE TITLE
Pearson correlation between each pair of genes in ALPACA modules

### ISSUE THREAD

**Issue by victortoledo** [netZoo/netZooR#227]
Hi everyone,

I am currently using the ALPACA and CONDOR functions in R and they return me the following objects: the DWBM values, row/col names, the membership of all TFs and genes and their scores.
However, I am failing to find a matrix of correlation between the genes, which would be important for me to plot the ALPACA modules. Since you reported in the original ALPACA article that it is possible to know the correlation between each pair of genes, I would like to ask for your help in this matter. How can I extract the Pearson correlation between each pair of genes in the ALPACA modules?

Thank you very much in advance.
Victor

**Comment by marouenbg** [comment]
Hi @victortoledo ,

So Alpaca does not compute correlation between the genes. To plot the modules, you can use the original network weights that you gave as input to CONDOR.

**Comment by victortoledo** [comment]
Thank you for answering @marouenbg, but I still don't know exactly what the original paper states with this sentence:

"The average absolute value of the Pearson correlation among all gene pairs in each module ranged only from 0.06 to 0.13, suggesting that network module detection does not merely reflect correlation-based clustering (Supplementary Fig. 3)."

"The average absolute value of the Pearson correlation coefficient among genes in the ALPACA modules ranged from 0.22 to 0.34 (Supplementary Fig. 3)."

Are these correlation coefficients based on a subset of genes in the original network according to modules defined by ALPACA or are these correlation coefficients defined by the correlation of the ALPACA matrix?

**Comment by johnquackenbush** [comment]
The correlation coefficients referred to here are pairwise correlation coefficients calculated using gene expression data. We calculated the correlations coefficients using gene expression data. We did this in response to questions from referees about what was driving cluster membership. The referee was questioning whether clusters were formed because eQTLs linked SNPs with genes having correlated patterns of expression. The analysis we reported indicated that, except for very small communities, the expression levels of genes were not highly correlated.

As you note, CONDOR does not rely in correlation in gene expression, but only SNP-gene cis- and trans-eQTL relationships.
